src/display.jl

```diff
@@ -355,7 +355,10 @@ function recordframe!(io::VideoStream)
 end
 
 """
-    save(path::String, io::VideoStream; framerate = 24)
+    save(
```

Comment:
Can we revert this style change? I prefer the one used before ;)

Reply:
I should really add a style guide for Makie & friends :(

Reply:
Would definitely help...I think I did this because there were too many kwargs, and didn't revert when all was done.